addons/settings/fnc_gui_settingList.sqf

```diff
@@ -19,6 +19,8 @@ private _lbData = [];
         _tooltip = localize _tooltip;
     };
 
+    _tooltip =  format ["%1 - %2", _x, _tooltip];
```

Comment:
```suggestion
    _tooltip =  format [["%1 - %2", "%1"] select (_tooltip isEqualTo ""), _x, _tooltip];
```
to remove ` - ` when tooltip is empty

Reply:
```suggestion
    if (_tooltip isEqualTo "") then {
        _tooltip = str _x;
    } else {
        _tooltip = _tooltip + endl + str _x;
    };
```

Reply:
commy2's suggestion looks more consistent with setting name tooltip